src/rpc/RPCHelpers.cpp

```diff
@@ -1674,23 +1683,38 @@ traverseTransactions(
             obj[JS(date)] = txnPlusMeta.date;
         }
         obj[JS(validated)] = true;
-
         txns.push_back(obj);
-    }
 
-    response[JS(ledger_index_min)] = minIndex;
-    response[JS(ledger_index_max)] = maxIndex;
+        if (!minReturnedIndex || txnPlusMeta.ledgerSequence < *minReturnedIndex)
+            minReturnedIndex = txnPlusMeta.ledgerSequence;
+        if (!maxReturnedIndex || txnPlusMeta.ledgerSequence > *maxReturnedIndex)
+            maxReturnedIndex = txnPlusMeta.ledgerSequence;
 
-    response[JS(transactions)] = txns;
+        /*! @brief We MOVED L1695-L1714 in the loop; wasn't in develop */
+        assert(cursor);
+        if (!forward)
+        {
+            response[JS(ledger_index_min)] = cursor->ledgerSequence;
+            response[JS(ledger_index_max)] = maxIndex;
+        }
+        else
+        {
+            response[JS(ledger_index_max)] = cursor->ledgerSequence;
+            response[JS(ledger_index_min)] = minIndex;
+        }
```

Comment:
Since all of this was removed from develop, I would just remove it here as well

Reply:
@cjcobb23 I should've elaborated further: those lines did not exist in RPCHelpers.cpp but were a part of the working AccountTx.cpp you approved in a prior PR before the merge conflict. (remember, as we moved any functionality from AccountTx.cpp into there). 

To replicate the same functionality based on the changes in develop, had to move these new lines into the loop.

Reply:
@cjcobb23 Update: Tested with suggested changes and successfully replicated same result 
![image](https://user-images.githubusercontent.com/47958858/191850229-6c1cd3bb-6cac-4d62-a5a3-5b1a1eeae0ec.png)